### ISSUE TITLE
Cannot upgrade to Rubygems 2.6.3

### ISSUE THREAD

**Issue by morgoth** [rubygems/rubygems#1579]
My current problem is that upgrading from 2.6.2 to 2.6.3 seems to not work:
```
wojtek@tp:~/Projects$ gem -v
2.6.2
wojtek@tp:~/Projects$ gem update --system
Updating rubygems-update
Successfully installed rubygems-update-2.6.3
Installing RubyGems 2.6.2
RubyGems 2.6.2 installed

=== 2.6.2 / 2016-03-12

Bug fixes:

* Fix wrong version of gem activation for bin stub. Pull request #1527 by
  Aaron Patterson.
* Speed up gem activation failures. Pull request #1539 by Aaron Patterson.
* Fix platform sorting in the resolver. Pull request #1542 by Samuel E.
  Giddins.
* Ensure we unlock the monitor even if try_activate throws. Pull request
  #1538 by Charles Oliver Nutter.



------------------------------------------------------------------------------

RubyGems installed the following executables:
	/home/wojtek/.rbenv/versions/2.3.0/bin/gem

RubyGems system software updated
wojtek@tp:~/Projects$ gem -v
2.6.2
```

This issue is related to:
  - [x ] The command line `gem`

**Comment by segiddins** [comment]
This is related to a bug in RUbyGems 2.6.2. Try `gem update --system=2.6.3`

**Comment by morgoth** [comment]
Thanks. That worked.

**Comment by monfresh** [comment]
I ran into this as well, and this closed issue was the only documentation I found to fix this. Would it be possible for `gem update --system` to detect that you are currently using 2.6.2, and if so, add the `=2.6.3` to allow you to upgrade?

I maintain a script that sets up a development environment on a Mac, and one of the things it runs is `gem update --system`. Setting the version number in the script is not a long term solution.

Thanks!

**Comment by segiddins** [comment]
@monfresh it's just a bug in 2.6.2

**Comment by monfresh** [comment]
Sorry, I had a brain fart. I'll add the logic to my script.

**Comment by morgoth** [comment]
@segiddins Just a note that I had exactly the same issue upgrading from 2.6.4 to 2.6.5

**Comment by segiddins** [comment]
@morgoth yup, I realized I didn't ship the fix in 2.6.5 about 2 minutes after releasing. I'll be doing a 2.6.6 when it isn't 2am (so in the morning)
